tensorflow/python/autograph/pyct/static_analysis/liveness.py

```diff
@@ -42,6 +43,18 @@ def __init__(self, graph, include_annotations):
 
   def init_state(self, _):
     return set()
+  
+  date="2023-09-23"
+  instructions=("Lambda fuctions will be no more assumed to be used in the" \
+               "statement where they are used, or at least in the same block." \
+               "https://github.com/tensorflow/tensorflow/issues/56089")
+  @deprecation.deprecated(date, instructions, warn_once=False)
```

Comment:
warn_once is probably useful to limit the verbosity

Reply:
Ok